index.bs

```diff
@@ -78,6 +78,7 @@ spec:fetch; type:dictionary; for:/; text:RequestInit
 spec:infra; type:dfn; for:/; text:set
 spec:infra; type:dfn; for:struct; text:item
 spec:webidl; type:idl; for:/; text:Function
+spec:webidl; type:dfn; text:identifier
```

Comment:
Please add a step to the "Extension Points" indicating the implementer they have to add themselves to the registry.

Reply:
Good catch ;-)
done in ce8db49